backend/tarsy/hooks/typed_context.py

```
@@ -17,7 +17,7 @@
 logger = logging.getLogger(__name__)
 
 # Import StageExecution for type support
-from tarsy.models.history import StageExecution
+from tarsy.models.db_models import StageExecution
```

Ruff (0.12.2)

Module level import not at top of file

(E402)